cvcl/templates/environment_definition_detail.html

```diff
@@ -5,12 +5,12 @@
     <h1>Environment Definition: {{ environment_definition.name }}
         <div class="pull-right">
             <a class="btn btn-warning" href="{% url 'envdefs.update' environment_definition.id %}" role="button">
-                {% bootstrap_icon "pencil" %} Update Environment Name</a>
+                {% bootstrap_icon "pencil" %} &nbsp;&nbsp;Update Environment Name </a>
```

Comment:
Adding `&nbsp;` is not the solution, it doesn't behave the way you expect in all cases.
Use margin in CSS instead.
Also you added a space before closing `a` tag.